demos/custom-tabs-example-app/src/main/java/org/chromium/customtabsdemos/PartialCustomTabActivity.java

```diff
@@ -171,7 +171,7 @@ private void openCustomTab() {
                 toolbarCornerRadiusPx);
         if (resizeBehavior != CustomTabsIntent.ACTIVITY_HEIGHT_DEFAULT) {
             customTabsIntent.intent.putExtra(
-                    CustomTabsIntent.EXTRA_ACTIVITY_RESIZE_BEHAVIOR, resizeBehavior);
+                    CustomTabsIntent.EXTRA_ACTIVITY_HEIGHT_RESIZE_BEHAVIOR, resizeBehavior);
```

Comment:
I don't know, but seems to make sense in case there's a possibility to resize WIDTH.

Reply:
I was surprised by the breaking change by renaming CustomTabsIntent.EXTRA_ACTIVITY_RESIZE_BEHAVIOR from 1.5 to 1.6 - it makes sense, just surprised (and I don't know enough about rules for breaking API changes in AndroidX).

Reply:
Yep, this is a breaking change because the project was using the 1.5.0-alpha01 and while in alpha it is expected for the API surface to still change. We ended up renaming this intent extra as part of the alpha cycle and it stabilized in beta.